app/src/debug/AndroidManifest.xml

```diff
@@ -124,12 +124,6 @@
             android:label="@string/ss_app_name"
             android:theme="@android:style/Theme.Translucent.NoTitleBar" />
 
-        <activity
-            android:name="com.thefinestartist.finestwebview.FinestWebViewActivity"
-            android:configChanges="keyboardHidden|orientation|screenSize"
-            android:screenOrientation="sensor"
-            android:theme="@style/FinestWebViewTheme.Light" />
-
```

Comment:
Removing this may cause the FinestWebView not launch - we use finest WebView to launch links within the browser app